gstools/covmodel/base.py

```diff
@@ -47,7 +46,7 @@
 HANKEL_DEFAULT = {"a": -1, "b": 1, "N": 200, "h": 0.001, "alt": True}
 
 
-class CovModel(metaclass=InitSubclassMeta):
+class CovModel(metaclass=type):
```

Comment:
@LSchueler is this the way to do it correctly?

Reply:
OK, metaclass was now redundant. We needed this, since `__init_subclass__` was added in py36: https://docs.python.org/3/reference/datamodel.html#object.__init_subclass__